package.json

```diff
@@ -36,11 +37,12 @@
     "pc-nrfconnect-devdep": "https://github.com/NordicSemiconductor/pc-nrfconnect-devdep.git#semver:^1.3.5"
   },
   "dependencies": {
-    "bootstrap": "4.1.0",
-    "react-bootstrap-slider": "2.1.4",
-    "electron-store": "^1.2.0",
+    "chart.js": "^2.8.0",
```

Comment:
I'm sure at least _chart.js_ and _react-..._ modules can be devDependencies, probably _nrf-dtm-js_ as well, since these will be webpacked to the final bundle.